admins/pageflow/entry.rb

```diff
@@ -169,23 +210,67 @@ def permitted_params
 
       private
 
+      def account_policy_scope
+        AccountPolicy::Scope.new(current_user, Pageflow::Account)
+      end
+
+      def theming_policy_scope
+        ThemingPolicy::Scope.new(current_user, Pageflow::Theming)
+      end
+
       def permitted_attributes
         result = [:title]
-
-        target = params[:id] ? resource : current_user.account
+        target = if !params[:id] && current_user.admin?
+                   Account.first
+                 elsif params[:id]
+                   resource
+                 else
+                   current_user.accounts.first
+                 end
         result += Pageflow.config_for(target).admin_form_inputs.permitted_attributes_for(:entry)
+        result += permit_account || []
+
+        if params[:id]
+          result << :folder_id if authorized?(:configure_folder_for, resource)
+        end
+
+        if params[:id]
+          accounts = resource.account
+        else
+          accounts = account_policy_scope.themings_accessible
+        end
+
+        result += permit_theming(accounts) || []
```

Comment:
Lets' rename these methods to `permitted_theming_attributes`/`permitted_account_attributes` and make them return an empty array if nothing is permitted.